packages/rule-sri/src/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * @fileoverview Require scripts and styles to use Subresource Integrity
+ */
+
+import * as sri from './sri';
```

Comment:
Maybe we should just make a convention that if there is a single rule in a package we use `rule.ts` for the name of the file?

Reply:
Done